TestLogs is flaky on Windows

```
2026-07-06T16:45:22.8837626Z === CONT  TestLogs/driver_json-file/without_tty/stdout_and_stderr
2026-07-06T16:45:22.8838796Z === CONT  TestLogs/driver_json-file/tty/only_stderr
2026-07-06T16:45:22.8839580Z === CONT  TestLogs/driver_json-file/without_tty/only_stderr
2026-07-06T16:45:25.9000611Z     logs_test.go:131: assertion failed: error is not nil: Error response from daemon: failed to create task for container: failed to create shim task: hcs::CreateComputeSystem d7bfcc654f9e659ec59c7eeb462643062832b6e9fc0b390df25c4db8f5218489: The parameter is incorrect.
2026-07-06T16:45:25.9027613Z --- FAIL: TestLogs/driver_json-file/without_tty/stdout_and_stderr (3.06s)
2026-07-06T16:45:25.9066742Z === CONT  TestLogs/driver_json-file/tty/only_stdout
2026-07-06T16:45:26.0136507Z     logs_test.go:131: assertion failed: error is not nil: Error response from daemon: failed to create task for container: failed to create shim task: hcs::CreateComputeSystem b36372143aae6e4bed50d1eded0854e35d4c612e62926b2b57bc4e21da631738: The parameter is incorrect.
2026-07-06T16:45:26.0146212Z --- FAIL: TestLogs/driver_json-file/without_tty/only_stderr (3.18s)
2026-07-06T16:45:26.0150663Z === CONT  TestLogs/driver_json-file/without_tty/only_stdout
2026-07-06T16:45:34.2837301Z     logs_test.go:184: Stripping: "\x1b[?9001h\x1b[?1004h\x1b[?25l\x1b[2J\x1b[m\x1b[Hthis is fine\x1b]0;C:\\bin\\sh.exe\a\x1b[?25haccidents happen\x1b[?9001l\x1b[?1004l"
2026-07-06T16:45:34.3093632Z --- PASS: TestLogs/driver_json-file/tty/stdout_and_stderr (11.48s)
2026-07-06T16:45:34.3999043Z     logs_test.go:184: Stripping: ""
2026-07-06T16:45:34.4636689Z --- PASS: TestLogs/driver_json-file/tty/only_stderr (11.63s)
2026-07-06T16:45:35.2996937Z     logs_test.go:184: Stripping: "\x1b[?9001h\x1b[?1004h\x1b[?25l\x1b[2J\x1b[m\x1b[Hthis is fine\x1b]0;C:\\bin\\sh.exe\a\x1b[?25haccidents happen\x1b[?9001l\x1b[?1004l"
2026-07-06T16:45:35.3458995Z --- PASS: TestLogs/driver_json-file/tty/only_stdout (9.46s)
2026-07-06T16:45:36.0757553Z --- PASS: TestLogs/driver_json-file/without_tty/only_stdout (10.02s)
2026-07-06T16:45:36.7045217Z --- FAIL: TestLogs/driver_json-file (0.46s)
2026-07-06T16:45:36.7396971Z --- FAIL: TestLogs (25.28s)
```

--- a/integration/container/logs_test.go
+++ b/integration/container/logs_test.go
@@ -132,6 +132,9 @@ func testLogs(t *testing.T, logDriver string) {
 				container.WithCmd("sh", "-c", "echo -n this is fine; echo -n accidents happen >&2"),
 				container.WithTty(tty),
 				container.WithLogDriver(logDriver),
+				// Logs do not need networking, and Windows HNS endpoint
+				// setup is sensitive to these parallel short-lived starts.
+				container.WithNetworkMode("none"),
 			)
 			defer apiClient.ContainerRemove(ctx, id, client.ContainerRemoveOptions{Force: true})
 

